server/db.js

```diff
@@ -1,11 +1,11 @@
-const Pool = require('pg').Pool;
+const { Pool } = require('pg');
 
 const pool = new Pool({
     user: 'postgres',
     password: 'postgres',
     host: 'localhost',
     port: 5433,
-    database: 'league'
+    // database: 'league'
```

Comment:
I had to remove this line locally to make the db work

Reply:
if u remove that line then u won't be connecting to the right database. it prob didn't work for u cuz u need to open postgres on ur side and create the database and the tables according to the db.sql file